geode-core/src/main/java/org/apache/geode/internal/InternalDataSerializer.java

```diff
@@ -252,6 +252,16 @@
   private static final CopyOnWriteHashMap<String, WeakReference<Class<?>>> classCache =
       LOAD_CLASS_EACH_TIME ? null : new CopyOnWriteHashMap<>();
   private static final Object cacheAccessLock = new Object();
+
+  private static final String PRE_GEODE_100_TCPSERVER_PACKAGE =
+      "com.gemstone.org.jgroups.stack.tcpserver";
+  private static final String POST_GEODE_100_TCPSERVER_PACKAGE =
+      "org.apache.geode.distributed.internal.tcpserver";
+  private static final String POST_GEODE_190_SERVER_CQIMPL =
+      "org.apache.geode.cq.internal.cache.query.ServerCQImpl";
+  private static final String PRE_GEODE_190_SERVER_CQIMPL =
```

Comment:
Always love to see these simple sorts of cleanups.

Reply:
👍